test/test_parse.py

```diff
@@ -671,6 +756,54 @@ def testExtract(text, expected_date, expected_leftover):
         testExtract("lets meet in 5seconds",
                     "2017-06-27 10:01:07", "lets meet")
 
+    def test_normalize_numbers(self):
+        self.assertEqual(normalize("remind me to do something at two to two"),
+                         "remind me to do something at 2 to 2")
+        self.assertEqual(normalize('what time will it be in two minutes'),
+                         'what time will it be in 2 minutes')
+        self.assertEqual(normalize('What time will it be in twenty two minutes'),
+                         'What time will it be in 22 minutes')
+        self.assertEqual(normalize("remind me to do something at twenty to two"),
+                         "remind me to do something at 20 to 2")
+
+        # TODO imperfect test, maybe should return 'my favorite numbers are 20 2',
+        #  let is pass for now since this is likely a STT issue if ever
+        #  encountered in the wild and is somewhat ambiguous, if this was
+        #  spoken by a human the result is what we expect, if in written form
+        #  it is ambiguous but could mean separate numbers
+        self.assertEqual(normalize('my favorite numbers are twenty 2'),
+                         'my favorite numbers are 22')
```

Comment:
may also be worth adding in where both numbers are digits 
> "my favorite numbers are 20 2"

Which I would expect would remain unchanged.

Reply:
added a test for this, but it is also changed. This shares logic with extract_number and will be hard to fix without a bigger refactor, if you are ok with that i would like to leave it for a follow up PR